### ISSUE TITLE
Add PSRQpy to astroquery

### ISSUE THREAD

**Issue by mattpitkin** [astropy/astroquery#1227]
I was wondering if you'd consider including my package [PSRQpy](https://psrqpy.readthedocs.io/en/latest/), used for querying the ATNF pulsar catalogue, into astroquery? If you think this is a good fit I'll get started on adding it and create a PR

One further question, if I moved it into astroquery would I still be able to easily do active development on it within astroquery?

**Comment by keflavich** [comment]
Yes, we'd welcome PSRQpy to astroquery.  At a glance, you could very nearly just incorporate it as a module directly, with a few small wrappers to match the rest of the astroquery interface.

And, yes, you can still actively develop when the package is in astroquery.  We do everything via pull requests, so we'd like you to have a good test suite.

We recently moved to a "continuous deployment" model, in which every pull request that is merged automatically triggers a new release.  This means that any changes you make via PR will be incorporated and available to users immediately, but of course, that also means we hold the PRs to a high standard and insist that they pass tests.

**Comment by mattpitkin** [comment]
Cool, I'll give this a go over the next few weeks and try and get it in decent shape for a PR. I already have a test suite that is run, so hopefully this may be enough, but I'd be happy to incorporate further tests.
